modules/infra/datapath/gr_mbuf.h

```diff
@@ -23,10 +23,13 @@ struct gr_trace_item {
 
 STAILQ_HEAD(gr_trace_head, gr_trace_item);
 
-#define GR_MBUF_PRIV_MAX_SIZE RTE_CACHE_LINE_MIN_SIZE
+#define GR_MBUF_PRIV_MAX_SIZE RTE_CACHE_LINE_MIN_SIZE * 2
```

Comment:
@rjarry 
As of now, the only common data is `struct iface *`, but this is enough to overflow with ipv6 related data structures.
The other option would be reuse a dynfield and create a specific accessor for it. Even if I'm not fan of that option, as it is specific, it avoids the increase of the PRIV_SIZE.
WDYT ?

Reply:
Honestly, I don't think it is a big deal to have 128 bytes of priv data. Unless we have proof that it has negative effects on performance, I would say we keep it like you changed it.

Reply:
ack, thanks !